libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -92,7 +92,7 @@ export const OcmClusterDetailsFormFields = ({
 
   const handleExternalPartnerIntegrationsChange = React.useCallback(
     (selectedPlatform: PlatformType) => {
-      const isOracleSelected = selectedPlatform === 'oci';
+      const isOracleSelected = selectedPlatform === 'external';
```

Comment:
Will the new API be expanded for other external platforms?
If so, I think we should make the code generic so that it will work for other platforms too.

We can also address that when the second "external" platform comes along.

Reply:
In the future users can add their own external platforms. We'll need an UX design to allow users to put platformName in case they want to use external platforms.
I think for htis first step we can do like this and we can make a ticket to followup the future changes.